opentelemetry-otlp/examples/basic-otlp/src/main.rs

```diff
@@ -52,12 +52,14 @@ fn init_metrics() -> metrics::Result<MeterProvider> {
 }
 
 fn init_logs() -> Result<opentelemetry_sdk::logs::Logger, LogError> {
+    let service_name = std::env::var("CARGO_BIN_NAME")
```

Comment:
sorry for the confusion earlier, but I meant use the standard [`env!`](https://doc.rust-lang.org/cargo/reference/environment-variables.html#environment-variables-cargo-sets-for-crates) to get this.  e.g. just
``` rust
let service_name = env!("CARGO_BIN_NAME");
```

Reply:
Yes my bad, this variable will not be available at run-time. Fixed it accordingly.